sound/soc/sof/sof-audio.c

```diff
@@ -301,6 +302,24 @@ static int sof_setup_pipeline_connections(struct snd_sof_dev *sdev,
 		}
 	}
 
+	/*
+	 * During route setup above, playback stream routes are set up by going
+	 * through sink path, capture stream routes are set up by going through
+	 * source path. Obviously, feedback route from a widget in the capture
+	 * stream to another widget in the playback stream is not able to be setup.
+	 *
+	 * The feedback route is setup here if widgets on both sides of the route
+	 * are active.
```

Comment:
useless comment, you are just saying the same thing as the commit message. Try to be clearer and more concise.
and don't use "obviously" please. This is far from obvious.